helpers.py

```diff
@@ -24,3 +24,47 @@ def sendVerificationEmail(mail, to, first_name, last_name, verification_token):
         return None
     except Exception as e:
         return e
+
+def sendApprovedEmail(mail, to, first_name, last_name):
+    # Create the callback URL
+    callback = ''
+    if app.config["ENV"] == 'development':
+        callback = os.getenv("FRONTEND_URL")
+    else:
+        callback = os.getenv("PROD_ROOT_URL")
+    callback = callback + 'auth/login'
```

Comment:
```suggestion
    callback = callback + 'login'
```
The login page route on the frontend will probably be `/login`